jax/_src/numpy/fft.py

```diff
@@ -126,15 +126,15 @@ def fftn(a: ArrayLike, s: Shape | None = None,
       supported.
 
   Returns:
-    An array containing the discrete Fourier transform of ``a``.
+    An array containing the multidimensional discrete Fourier transform of ``a``.
 
   See also:
-    - :func:`jax.numpy.fft.fft`: Computes the discrete Fourier transform of
-      one-dimensional array.
-    - :func:`jax.numpy.fft.ifft`: Computes the inverse of discrete Fourier transform
-      of one-dimensional array.
-    - :func:`jax.numpy.fft.ifftn`: Computes the inverse of discrete Fourier transform
-      of multidimensional array.
+    - :func:`jax.numpy.fft.fft`: Computes one-dimensional discrete Fourier
```

Comment:
```suggestion
    - :func:`jax.numpy.fft.fft`: Computes a one-dimensional discrete Fourier
```

And the ones below... 👇